app/src/main/java/de/westnordost/streetcomplete/util/LastPickedValuesStore.kt

```diff
@@ -28,11 +28,11 @@ class LastPickedValuesStore<T : Any>(
     private fun getKey() = Prefs.LAST_PICKED_PREFIX + key
 }
 
-/* Returns the `target` most-common non-null items in the first `historyCount`
+/** Returns the [target] most-common non-null items in the first [historyCount]
```

Comment:
```suggestion
/**
 * Returns the [target] most-common non-null items in the first [historyCount]
```
Or indent the other lines below to match